libs/subliminal_patch/providers/titulky.py

```diff
@@ -62,152 +46,62 @@ def __init__(self,
                  sub_id,
                  imdb_id,
                  language,
-                 names,
                  season,
                  episode,
-                 year,
-                 releases,
-                 fps,
+                 release_info,
                  uploader,
                  approved,
                  page_link,
                  download_link,
-                 skip_wrong_fps=False,
                  asked_for_episode=None):
         super().__init__(language, page_link=page_link)
 
-        self.names = names
-        self.year = year
         self.sub_id = sub_id
         self.imdb_id = imdb_id
-        self.fps = fps
         self.season = season
         self.episode = episode
-        self.releases = releases
-        self.release_info = ', '.join(releases)
+        self.releases = [release_info]
+        self.release_info = release_info
         self.language = language
         self.approved = approved
         self.page_link = page_link
         self.uploader = uploader
         self.download_link = download_link
-        self.skip_wrong_fps = skip_wrong_fps
         self.asked_for_episode = asked_for_episode
         self.matches = None
 
-        # Try to parse S00E00 string from the main subtitle name
-        season_episode_string = None
-        if len(self.names) > 0:
-            season_episode_string = re.findall(r'S(\d+)E(\d+)', self.names[0], re.IGNORECASE)
-
-        # If we did not search for subtitles with season and episode numbers in search query,
-        # try to parse it from the main subtitle name that most likely contains it
-        if season_episode_string:
-            if self.season is None:
-                self.season = int(season_episode_string[0][0])
-            if self.episode is None:
-                self.episode = int(season_episode_string[0][1])
-
     @property
     def id(self):
         return self.sub_id
 
-    def get_fps(self):
-        return self.fps
-
     def get_matches(self, video):
         matches = set()
-        _type = 'movie' if isinstance(video, Movie) else 'episode'
-        # Subtitle's names (could be series/episode/movie name) present in the subtitle details page
-        # Consists of the main name and alternative names, stripped of the S00E00 substring
-        sub_names = self._remove_season_episode_string(self.names)
-
-        if _type == 'episode':
-            # EPISODE
+        media_type = 'movie' if isinstance(video, Movie) else 'episode'
 
+        if media_type == 'episode':
             # match imdb_id of a series
             if video.series_imdb_id and video.series_imdb_id == self.imdb_id:
-                matches.add('series')
-                matches.add('series_imdb_id')
+                matches |= {'series_imdb_id', 'series', 'year'}
```

Comment:
Can you please change this ?  `|=` is for python 3.9+, we support 3.7+ so this will cause some issues for older python installations

Reply:
Oh, I did not realize. I'll make a new PR fixing this today. Thanks!

Reply:
I looked into it some more, and the syntax is supported for sets in python 3.7+ ([docs](https://docs.python.org/3.7/library/stdtypes.html#frozenset.update)). You are probably referring to `|=` operator overload for dicts that was introduced in 3.9.